orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/PipelineStarterListener.groovy

```diff
@@ -76,11 +81,13 @@ class PipelineStarterListener implements JobExecutionListener {
             def queuedExecution = executionRepository.retrievePipeline(id)
             log.info("starting pipeline ${nextPipelineId} due to ${execution.id} ending")
             pipelineStarter.startExecution(queuedExecution)
-          } else {
+            startTracker.removeFromQueue(execution.pipelineConfigId, id)
```

Comment:
@tomaslin yeah, thanks, that makes sense.  you are suggesting to have a UI checkbox in the individual pipeline configuration level for this, correct?